README.md

```diff
@@ -35,17 +35,17 @@ It is recommended to install these dependencies using the [robotology-superbuild
     $ conda install mamba
     $ conda create -n robotologyenv
     $ conda activate robotologyenv
-    $ mamba install -c robotology iDynTree qpOASES icub-models wb-toolbox whole-body-controllers
+    $ mamba install -c robotology YARP yarp-matlab-bindings iDynTree qpOASES icub-models wb-toolbox whole-body-controllers
```

Comment:
Conda package names are lowercases, not sure if YARP or iDynTree also works, but I would stick to lowercase to ensure that we do not have any problem in some corner case.

Reply:
Absolutely. I just looked at the `conda list`. Anyway there is a bug in the CMakeLists.txt due to the change in the location of the folder `RobotDynamicsWithContacts` in the source repo, so I'll fix also the README and the script for MATLAB online.

Actually, I shouldn't have to specify `yarp` in the list of packages, since it is a dependency of `yarp-matlab-bindings` right?
(I don't do it in the actual script)

Reply:
> Actually, I shouldn't have to specify yarp in the list of packages, since it is a dependency of yarp-matlab-bindings right?
(I don't do it in the actual script)

Yes, that is correct!

Reply:
I kept `yarp` in the list for the sake of clarity.

Reply:
I removed it again.